mindjunkies/forums/tests/test_views.py

```
@@ -0,0 +1,369 @@
+import pytest
+from django.test import TestCase, RequestFactory, Client
```

Ruff (0.8.2)

`django.test.Client` imported but unused

Remove unused import: `django.test.Client`

(F401)